### ISSUE TITLE
UI: Filter Dialog may benefit from fine tuning

### ISSUE THREAD

**Issue by ralfstrandell** [jariarkko/psgeo#113]
_Main Filter Window_ (upper left)
This is quite small > hence the need for an extended window > hence split in two filter dialog > not fun :(

_Extended Filter Window_
This one clouds the view and cannot be moved around on the map > not fun :(

**Possible solution**: consider a sidepanel with width 0px/250px set by script (filter button, onclick, switch 0->250, default 0)
**Possible solution**: expand/collapse button that shows/hides more stuff (and adjusts the window size accordingly): small box/big box
**Possible solution**: anything that keeps everything in the same dialog really

**Comment by jariarkko** [comment]
We can experiment with this, but i'd like to see how well it works and and what it feels like before switching. And even if the side panel is there its contents need to be dynamic, because the set of activities and their searchable properties should be data-driven, and is likely to change often.

**Comment by ralfstrandell** [comment]
It would only replace Google Maps InfoWindow as a container. Contents could still be generated with existing code: just se innerhtml to put it in. But don't hold your breath, I am not working on this now.

**Comment by ralfstrandell** [comment]
I will work on this. It will be in another branch, and it will be an optional UI. Then we can evaluate its usefullness. Expect some trials in a week or so. All dynamical features shall remain. Things are still placed in divs with id's.

**Comment by ralfstrandell** [comment]
This has now been solved in a backwards compatible manner.

classicUI=true in psgeoDomainDefaults:
This keeps the old user interface with filtering options in a Google Maps infoWindow.
If the window has too much content and overruns the screen border, then autopanning creates an unpleasant effect.

classicUI=false
A Google Maps custom control opens at BOTTOM_CENTER
The style (color, border, margin, padding, width, ...) can be set in css/style.css
This dialog can now easily be finetuned for better aesthetics.
I've tested this and will now commence finetuning the looks.

Hence, I consider this ticket worth closing.
Awaiting comments.
